src/DonationForms/AsyncData/Actions/GetAsyncFormDataForListView.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Give\DonationForms\AsyncData\Actions;
+
+use Give\DonationForms\AsyncData\AdminFormListView\AdminFormListViewOptions;
+use Give\DonationForms\AsyncData\AsyncDataHelpers;
+use Give\DonationForms\AsyncData\FormGrid\FormGridViewOptions;
+
+/**
+ * @unreleased
+ */
+class GetAsyncFormDataForListView
+{
+    /**
+     * @unreleased
+     */
+    public function __invoke()
+    {
+        $options = give_clean($_GET);
+
+        if ( ! isset($options['formId'])) {
```

Comment:
We should also validate that the `formId` is a DonationForm (and not just an arbitrary post ID or something else)

Reply:
Resolved in ea887b14a050fc16f2c4856cf7eebe8c49d022d7